Web/HTML1/MetaGurukul.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+	<form>
+	<fieldset form="">
+		<table>
+		<tr><td align="right"><button type="button">Help</td></tr>
+			<tr>
+				<td colspan=2>Metagurukul Account Registration</td>
+			</tr>
+			<tr>
+				<td align="right">Name:</td>
+				<td><input type="text" placeholder="First Name"><input
+					type="text" placeholder="Last Name"></td>
+			</tr>
+
+			<tr>
+				<td align="right">Gender:</td>
+				<td><select><option>- Select One -</option></select></td>
```

Comment:
ACTION:Screenshot is for reference of functionality but we have implement the whole functionality. Please add male,female select option.